### ISSUE TITLE
: or * in header value crash site. Lexer skips out.

### ISSUE THREAD

**Issue by serrynaimo** [vercel/next.js#15580]
# Bug report
`next.config.js` header configuration
```
async headers () {
    return [{
      source: '/:path*',
      headers: [{
        key: 'Content-Security-Policy',
        value: 'default-src https:',
      }]
}
```

results in 
```
Missing parameter name at 172
/node_modules/next/dist/compiled/path-to-regexp/index.js:47:23
```
when accessing site

## Describe the bug
The : or * characters in header value seem to bug the lexer

## Expected behavior
Header should be set as described

## System information
- Version of Next.js: 9.5.0

**Comment by Timer** [comment]
This was just fixed in 9.5.1, please upgrade!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
